components/Button/Button.styles.js

```diff
@@ -0,0 +1,15 @@
+import styled, { css } from 'styled-components';
+import { pxToRem } from '../_theme';
+
+const StyledButton = styled.button`
+  background: linear-gradient(to left,${props => props.theme.variables.colors.primary.default} 50%,${props => props.theme.variables.colors.secondary.default});
+  border-radius: ${props => props.theme.borderRadius.small};
+  color: ${props => props.theme.variables.colors.grayScale.white};
+  display: inline-block;
+  font-size: ${pxToRem(16)};  
```

Comment:
We should be pulling `font-size` from _theme.js. There's a `typography.body.size[default || small || subtitle]`